apps/kyverno-guardrails/charts/st-charts.yaml

```diff
@@ -0,0 +1,7 @@
+st-charts:
```

Comment:
FYI, I created chart-ctl.py script to generate a wrapper chart and service-template chart automatically. It's not prepared to accept also custom chart as used here (kyverno-guardrails-0.1.0). Once you have the kyverno-guardrails chart prepared in PR I can take over and adjust the script so we can use it for service-template validation even here, ok?

Reply:
😄 ok

Reply:
@BROngineer done, please review: https://github.com/k0rdent/catalog/pull/351